README.md

```diff
@@ -37,57 +37,94 @@ Example:
 ```javascript
 var gplay = require('google-play-scraper');
 
-gplay.app({appId: 'com.dxco.pandavszombies'})
+gplay.app({appId: 'com.google.android.apps.translate'})
   .then(console.log, console.log);
 ```
 Results:
 
 ```javascript
-{ appId: 'com.dxco.pandavszombies',
-  url: 'https://play.google.com/store/apps/details?id=com.dxco.pandavszombies&hl=en&gl=us'
-  title: 'Panda vs Zombies',
-  description: 'Panda, my friend, Panda is the answer. But not any Panda: Rocky the Panda!',
-  descriptionHTML: 'Panda, my friend, Panda is the answer. But not any Panda: <b>Rocky the Panda!</b>',
-  summary: 'Help Rocky the Panda warrior to fight zombie games and save the Panda kind.',
-  installs: '10,000+',
-  minInstalls: 10000,
-  score: 4.5025907,
+{
+  title: 'Google Translate',
+  description: '• Translate between 103 languages by typing\r\n' +
```

Comment:
why those '+' between strings? this is supposed to be the actual output returned by the node shell, and if it's too long just cut it or add a ...

Reply:
![cmd_2020-01-27_12-52-13](https://user-images.githubusercontent.com/614159/73212859-f509f800-4103-11ea-86df-4d8eb55248da.png)

To answer your first question, it is the app method that gives the + strings, unless I'm using the API in a wrong way.

But I agree the example will be lengthy. Let me change it.